src/main/java/training/TableItems.java

```diff
@@ -0,0 +1,28 @@
+package training;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.dynamodbv2.document.PutItemOutcome;
+import com.amazonaws.services.dynamodbv2.document.Table;
+
+public class TableItems {
+    public void putItem() {
+
+        AmazonDynamoDB client = AmazonDynamoDBClientBuilder.standard().build();

```

Comment:
Is this working? I mean does it insert items?

Reply:
yes

Reply:
I'm also confused how its working without accesskey and secretkey

Reply:
Please delete any items in the table and then run

Reply:
still its working

Reply:
Ok good

Reply:
but im confused how its working. i even made a new project and i copied this class and the app class code in that and it still worked why? without access key and secret key how?